packages/evo-marko/src/tags/evo-accordion/examples/autoCollapsed.marko

```diff
@@ -0,0 +1,20 @@
+<evo-accordion ...input autoCollapse=true>
```

Comment:
Lets change these examples to use the new API for showing that it works. 
Basically `autoCollapse` should not be used anymore
We should pass in `open=-1` which will treat it as auto collapse=true